docs/pages/faq.rst

```diff
@@ -0,0 +1,61 @@
+Frequently Asked Question
+=========================
+
+
+Why doesn't :class:`~mcstatus.server.BedrockServer` have an async :meth:`~mcstatus.server.MCServer.lookup` method?
+------------------------------------------------------------------------------------------------------------------
+
+.. this title looks ok in rendered docs
```

Comment:
This comment is unnecessary and can be removed, people will be able to just look at the docs if they think it doesn't look good, and I wouldn't immediately assume that a title like this would look bad anyway.

Reply:
Removed in [last force-push](https://github.com/py-mine/mcstatus/compare/9086ff3d9484d8cde97d4439741329a1a487017d..f71fea6b2261db29714c612dcf737c7c75465482), thanks!